src/ngamsPlugIns/ngamsPlugIns/ngamsSdmMultipart.py

```diff
@@ -58,168 +62,151 @@ def specificTreatment(fo):
-        fileVersion, relPath, relFilename,\
-                     complFilename, fileExists =\
-                     ngamsPlugInApi.genFileInfo(srvObj.getDb(),
-                                                srvObj.getCfg(),
-                                                reqPropsObj, diskInfo,
-                                                stagingFilename, fileId,
-                                                finalName, [date])
-
-        # Generate status.
-        logger.debug("Generating status ...")
-        if not format:
-            format = ngamsPlugInApi.determineMimeType(srvObj.getCfg(),
-                                                  stagingFilename)
-        fileSize = ngamsPlugInApi.getFileSize(stagingFilename)
-        return ngamsPlugInApi.genDapiSuccessStat(diskInfo.getDiskId(),
-                                                 relFilename,
-                                                 fileId, fileVersion, format,
-                                                 fileSize, uncomprSize,
-                                                 compression, relPath,
-                                                 diskInfo.getSlotId(),
-                                                 fileExists, complFilename)
-    except Exception as err:
-        # mbauhofe: replaced missing variables
-        # old code:
-        # errMsg = genLog("NGAMS_ER_DAPI_BAD_FILE", [os.path.basename(filename),
-        #                                            _PLUGIN_ID, err])
-
-        errMsg = genLog("NGAMS_ER_DAPI_BAD_FILE",
-                        [os.path.basename(stagingFilename),
-                         _PLUGIN_ID, str(err)])
-        raise Exception(errMsg)
-
-if __name__ == "__main__":
-    import sys
-    if len(sys.argv) < 2:
-        print("Usage: ngamsSdmMultipart.py <test_file>")
-        sys.exit()
-    try:
-        fo = open(sys.argv[1],'r')
-        (file_id,fileName, type) = specificTreatment(fo)
-        print(file_id, fileName, type)
-    except:
-        raise
 
+        # logger.info("Compressing file %s using %s compression",
+        #             staging_filename, compression)
+        # exit_code, std_out = ngamsPlugInApi.execCmd("{0} {1}".format(
+        #     compression, staging_filename))
+        # if exit_code != 0:
+        #     error_message = _PLUGIN_ID + ": Problems during archiving! " + \
+        #                     "Compressing the file failed"
+        #     raise Exception(error_message)
+        # staging_filename = staging_filename + ".Z"
+        # logger.info("Finished compressing staging file %s", staging_filename)
+
+        # Remember to update the temporary file name in the request
+        # properties object
+        request_object.setStagingFilename(staging_filename)
+
+        # # ToDo: Handling of non-existing fileId
+        # if file_id == -1:
+        #     file_id = ngamsPlugInApi.genNgasId(server_object.getCfg())
+
+        today = date.today().isoformat()
```

Comment:
I'm not exactly sure if there is any difference, but you might want to consider using `ngamsLib.ngamsCore.toiso8601` here instead. `today` seems to be used just to create a directory name so the actual value might be irrelevant, but I'm rusty on this. If anything else, it would make the code align better with the rest of NGAS, where `toiso8601` and `fromiso8601` are used almost uniformly for time/string formatting/parsing.

Reply:
@awicenec - In this scenario we I believe we only need the date ('YYYY-MM-DD' format). I checked this on an old NGAS python shell.
```
>>> from mx import DateTime
>>> DateTime.now().date
'2020-11-11'
```

Reply:
@rtobar - Your suggestion looks good but if you read my previous comment I think we only need 'YYYY-MM-DD'. I checked `ngamsLib.ngamsCore.toiso8601` which gives us the date/time to milliseconds resolution.

Reply:
There is an extra `fmt` parameter in `toiso8601` that allows you to specify one of the pre-defined formats, of which `FMT_DATE_ONLY` gives you that exactly. In any case this is a small detail, and more importantly the resulting will be the same, so